README.md

```diff
@@ -4,6 +4,11 @@ Sets up a new [Cloak](https://github.com/BKWLD/cloak) (Nuxt + CMS) based project
 
 ![](https://i.pinimg.com/originals/75/af/04/75af04c5f9fa6e26231640f7d368f042.gif)
 
+## Install
+
+If installing fresh, or upgrading from version 2
+`yarn global add create-cloak-app@^3`
+
```

Comment:
When did you run into needing to do this?  I _would_ have thoght `yarn create cloak-app` would always use the latest version, even it was a major point difference...

Reply:
oh, I assumed that since I never explicitly updated, it would use whatever version I had globally, which would have been 2. If that just works, then there's no need for this